tests/test_widgets.py

```diff
@@ -86,13 +86,13 @@ def SomeComponent():
     input_1 = await display.page.wait_for_selector("#i_1")
     input_2 = await display.page.wait_for_selector("#i_2")
 
-    await input_1.type("hello", delay=20)
+    await input_1.type("hello", delay=50)
```

Comment:
I recommend setting this to 250ms, and storing it within a global.

Reply:
Also created an issue to be able to set a default delay https://github.com/microsoft/playwright-python/issues/1614